code/game/machinery/computer/communications.dm

```diff
@@ -16,20 +15,23 @@
 	req_access = list(ACCESS_HEADS)
 	circuit = /obj/item/circuitboard/communications
 	var/prints_intercept = 1
```

Comment:
Not part of your edits, but could you make these defaults `TRUE`/`FALSE` to aid readability